ts/screens/authentication/cie/CieUnexpectedErrorScreen.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * A screen to alert the user about the number of attempts remains
+ */
+import * as React from "react";
+import I18n from "../../../i18n";
+import { useIONavigation } from "../../../navigation/params/AppParamsList";
+import ROUTES from "../../../navigation/routes";
+import { OperationResultScreenContent } from "../../../components/screens/OperationResultScreenContent";
+
+const CieUnexpectedErrorScreen = () => {
+  const navigation = useIONavigation();
+
+  const navigateToCiePinScreen = React.useCallback(() => {
+    navigation.navigate(ROUTES.AUTHENTICATION, {
+      screen: ROUTES.CIE_PIN_SCREEN
+    });
+  }, [navigation]);
+
+  const navigateToAuthenticationScreen = React.useCallback(() => {
+    navigation.reset({
```

Comment:
In app there are few occurrences of navigation.reset (even in similar situations) do you think it is appropriate to use this feature and also ask other developers to make it a best practice?

Reply:
> do you think it is appropriate to use this feature
Using `reset` depends on the specific flow/use case. Here I only extracted the navigation from the deprecated use of [`NavigationService`](https://github.com/pagopa/io-app/blob/17fdb279af0b314414bf313452eb3c6fdf9adc69/ts/store/actions/navigation.ts#L33) in a JSX Component, where we should use `navigation`, coming from `useNavigation()`, instead.
> ask other developers to make it a best practice
I don't. As I said before using `reset` instead of other kind of navigations depends on the specific use case/flow. Remember that we also have an architecture with multiple navigation stack, where the main stack changes according to some conditions, so I don't think that `reset` would be applicable in a more generic way, but in this specific case, reset works great.

I don't know if I've answered your doubts. What do you think?